Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -174,80 +183,22 @@ - (BOOL)retry
     // request might have succeeded on the remote server, and by retrying we'd be issuing it again.
     // PUT and POST requests aren't generally idempotent, but the ones sent by the replicator are.
 
-    if (_retryCount >= kMaxRetries) return NO;
-    NSTimeInterval delay = RetryDelay(_retryCount);
-    ++_retryCount;
-    CDTLogVerbose(CDTTD_REMOTE_REQUEST_CONTEXT, @"%@: Will retry in %g sec", self, delay);
-    [self startAfterDelay:delay];
-    return YES;
+//    if (_retryCount >= kMaxRetries) return NO;
```

Comment:
This feels like it was commented out just because it didn't work. Why doesn't it work?